editor/src/components/documentation/getting-started.tsx

```diff
@@ -222,7 +222,7 @@ export const GettingStarted = betterReactMemo('Getting Started', () => {
       <FeaturedProjects />
 
       <H1>Keyboard Shortcuts</H1>
-      <P>
```

Comment:
the styled component ([link](https://github.com/concrete-utopia/utopia/blob/c773a0b171374d43aa0c3b0d5d062b330d14b26f/editor%2Fsrc%2Fcomponents%2Fdocumentation%2Fdocumentation-components.tsx#L51-L53)) has a marginTop. We could just make sure _all_ the "paragraph" (`p`, `P`, et. al.) use `P`, and change the `P` styled component to extend a `div` instead, which would fix the invalid nesting problem.

Reply:
Ok good catch, seems we want to have all of these using that `marginTop`, but making this change means we're losing any possible future legitimate use of `P` as a paragraph tag. If we're going to make this change, we should just change the name of that styled component too so that it's not misleading (I don't care what name we go with, as long as it's not "P").